birdhouse/components/stac/.gitignore

```diff
@@ -1,2 +1 @@
-conf.extra-service.d/stac.conf
-config/magpie/config.yml
```

Comment:
The files generated from the templates should remain listed here, to avoid them being re-added back.